dartagnan/src/main/java/com/dat3m/dartagnan/configuration/Property.java

```diff
@@ -59,8 +59,9 @@ public BooleanFormula getSMTVariable(EncodingContext ctx) {
 
     public BooleanFormula getSMTVariable(Axiom ax, EncodingContext ctx) {
         Preconditions.checkState(this == CAT_SPEC);
-        return ctx.getBooleanFormulaManager()
-                .makeVariable("Flag " + Optional.ofNullable(ax.getName()).orElse(ax.getRelation().getNameOrTerm()));
+        final String varName = ctx.getFormulaManager().escape("Flag " +
+                Optional.ofNullable(ax.getName()).orElse(ax.getRelation().getNameOrTerm()));
+        return ctx.getBooleanFormulaManager().makeVariable(varName);
```

Comment:
Is there any of our examples where the escape is really necessary?

Reply:
Yes and no. There are axioms in Spirv like `flag ~empty (locord \ loc) as AssertLocordSameLoc` which only worked so far because they are named. If you wrote `flag ~empty (locord \ loc)` instead, you would run into an exception.
I noticed this just because some memory model processing failed to preserve axiom names.